jwala-integration-test/src/test/resources/com/cerner/jwala/ui/selenium/configuration/pagination.feature

```diff
@@ -6,273 +6,274 @@ Feature: Pagination
     And I am in the group tab
     And I created a group with the name "group1"
     And I created a group with the name "group2"
+    And I load properties file
     And I am in the media tab
     And I created a media with the following parameters:
       | mediaName       | apache-httpd-2.4.20     |
       | mediaType       | Apache HTTPD            |
       | archiveFilename | apache-httpd-2.4.20.zip |
-      | remoteDir       | c:\ctp                  |
+      | remoteDir       | remoteDir               |
     And I load properties file
     And I am in the web server tab
     And I select the dropdown of "webserver" with option "100"
     And I created a web server with the following parameters:
-      | webserverName      | 1                   |
+      | webserverName      | 1WebServer          |
```

Comment:
WebServer1, WebServer2, ... it more conventional. Judging from the name, attaching the WebServer part from the number required less effort hence the unconventional names. In addition the groups were renamed to Group1, Group2 etc, we should always be consistent on our naming convention.